### PR TITLE
consul quick start on docker

### PR DESCRIPTION
**Pull request by rshriram**

### CODE REVIEW

File: _docs/setup/consul/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    ```bash
+    istioctl context-create --api-server http://172.28.0.13:8080
+    ```
+
+1. Change directory to the root of the Istio installation directory.
+
+1. Bring up the Istio control plane containers:
+
+    ```bash
+    docker-compose -f samples/bookinfo/consul/control-plane.yaml up -d
```
Comment (frankbu) [inline _docs/setup/consul/quick-start.md]: Should this be moved to `install/consul/control-plane.yaml`?

File: _docs/setup/consul/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```bash
+docker-compose -f <your-app-spec>.yaml up -d)
+```
+
+## Uninstalling
+
+1. Uninstall Istio core components by removing the docker containers:
+
+```bash
+docker-compose -f samples/bookinfo/consul/control-plane.yaml down
```
Comment (frankbu) [inline _docs/setup/consul/quick-start.md]: same as above

File: _docs/setup/consul/quick-start.md (hunk 1 above)
Comment (rshriram) [reply _docs/setup/consul/quick-start.md]: Yes.. I was thinking the same. Requires a separate PR in istio/istio

File: _docs/setup/consul/quick-start.md (hunk 2 above)
Comment (rshriram) [reply _docs/setup/consul/quick-start.md]: ack. Working on the PR

**Pull request merged**